e2e-tests/tests/reserve/conftest.py

```diff
@@ -0,0 +1,188 @@
+from pytest import fixture
+from config.api_config import ApiConfig
+from src.blockchain_api import BlockchainApi
+from src.cardano_cli import cbor_to_bech32, hex_to_bech32
+from src.partner_chains_node.models import Reserve, VFunction
+import json
+import logging
+
+
+@fixture(scope="session")
+def governance_address(config: ApiConfig) -> str:
+    return config.nodes_config.governance_authority.mainchain_address
+
+
+@fixture(scope="session")
+def payment_key(config: ApiConfig, governance_skey_with_cli):
+    return config.nodes_config.governance_authority.mainchain_key
+
+
+@fixture(scope="session")
+def governance_vkey_bech32(config: ApiConfig):
+    vkey = config.nodes_config.governance_authority.mainchain_pub_key
+    vkey_bech32 = hex_to_bech32(vkey, "addr_vk")
+    return vkey_bech32
+
+
+@fixture(scope="package")
+def reserve(reserve_token, v_function: VFunction) -> Reserve:
+    reserve = Reserve(token=reserve_token, v_function=v_function)
+    return reserve
+
+
+@fixture(scope="package")
+def v_function(v_function_factory, config: ApiConfig):
+    v_function_path = config.nodes_config.reserve.v_function_script_path
+    v_function = v_function_factory(v_function_path)
+    return v_function
+
+
+@fixture(scope="package")
+def minting_policy_filepath(api: BlockchainApi, governance_vkey_bech32, write_file):
+    key_hash = api.cardano_cli.get_address_key_hash(governance_vkey_bech32)
```

Comment:
I don't understand it. `governance_vkey_bech32` seems to be a string starting with `addr_vk`. But `get_address_key_hash` expects a file.

Reply:
`governance_vkey_bech32` is indeed a string, however it's not starting with `addr_vk` The prefix is bech32 HRP which is needed to specify the type of encoding.

`api.cardano_cli.get_address_key_hash` expects `--payment-verification-key`, not `--payment-verification-key-file`